src/Microsoft.PowerShell.Commands.Utility/commands/utility/Get-Error.cs

```diff
@@ -15,6 +15,7 @@ namespace Microsoft.PowerShell.Commands
     [Cmdlet(VerbsCommon.Get, "Error",
         HelpUri = "https://docs.microsoft.com/powershell/module/microsoft.powershell.utility/get-error?view=powershell-7&WT.mc_id=ps-gethelp",
         DefaultParameterSetName = NewestParameterSetName)]
+    [OutputType(typeof(ErrorRecord), typeof(Exception))]
```

Comment:
Not sure if this is valid
```suggestion
    [OutputType('PSExtendedError', typeof(ErrorRecord), typeof(Exception))]
```

Reply:
This is C# so you need double quotes, but otherwise I believe that's perfectly valid :)

Reply:
`OutputType()` takes types, not strings.  `PSExtendedError` is a fake type applied to ErrorRecords and Exceptions.

Reply:
![image](https://user-images.githubusercontent.com/32407840/67248406-61ed6200-f432-11e9-8266-4a769238ffeb.png)

It'll do both, though I'm not sure if PSTypeNames or fake types like that "work". 🙂

Reply:
Hmm, I guess I can just say it returns PSExtendedError since I explicitly remove the ErrorRecord and Exception typenames so they don't get priority in rendering.  Will update.